tests/Integration/Database/EloquentModelScopeTest.php

```diff
@@ -27,4 +35,10 @@ public function scopeExists()
     {
         return true;
     }
+
+    #[NamedScope]
+    public function existsAsWell()
```

Comment:
Need to also test if child model can utilise attribute named scope defined in parent model.

method need to be updated to protected and contains parameter as example to real usage

Reply:
I'm afraid, I can't follow you? Can you elaborate on that?

Reply:
![CleanShot 2025-02-05 at 19 48 00](https://github.com/user-attachments/assets/1b4ed1ec-4d8a-4183-9b46-b3549e607cd2)

This would be the actual use case, correct?